src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar top-search-bar"
+      data-qa="big"
+    >
+      <input
+        class="search-bar-input search-bar-input--top"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar bottom-search-bar"
+      data-qa="small"
```

Comment:
Class names represent the meaning of the content (not the styles or tag names). In your case, 'top-search-bar' and 'bottom-search-bar' don't provide any semantic meaning. Try to use names that describe the function or content, like 'primary-search-bar' and 'secondary-search-bar'.